src/features/wallet/Wallet.tsx

```diff
@@ -163,10 +169,24 @@ const Wallet: FC<Props> = () => {
             <p className="wallet-title">Voting Wallet</p>
             {isConnected && votingAddress ? (
               <>
-                <VotingAddress>{shortenAddress(votingAddress)}</VotingAddress>
-                <Connected>
-                  Connected with {onboard?.getState().wallet.name}
-                </Connected>
+                <VotingAddress>
+                  {hotAddress ? "DVC: " : null} {shortenAddress(votingAddress)}
+                </VotingAddress>
+                {hotAddress ? (
+                  <VotingAddress>
+                    Voting: {shortenAddress(hotAddress)}
+                  </VotingAddress>
+                ) : null}
+                {signingPK ? (
+                  <Connected>
+                    Connected with {onboard?.getState().wallet.name}{" "}
```

Comment:
is `wallet.name` always going to exist? i feel like this could cause a crash, maybe safer to use optional chaining? wdyt

Reply:
I'm not sure -- Onboard's docs are not fully clear on this.

Reply:
We could have a fallback name if you'd like.

Reply:
its les about a fallback name and more about indexing into an undefined.  id probably just want to check `getState()?.wallet?.name`